### DISCUSSION TITLE
Module not found: Can't resolve '@next-translate-root/locales'

### DISCUSSION THREAD

**Discussion by mekhamata** [vinissimus/next-translate#898] [Q&A] [answered]
hello, i installed the next-translate, but i have this problem:

`Module not found: Can't resolve '@next-translate-root/locales'
  66 |       isLoader: true,
  67 |       skipInitialProps: true,
> 68 |       loadLocaleFrom: (l, n) => import(`@next-translate-root/locales/${l}/${n}`).then(m => m.default),
     |                                ^
  69 |     });`

this is my next.config.js:
`/** @type {import('next').NextConfig} */
// const { i18n } = require('./next-i18next.config');
// const nextTranslate = require('next-translate');
const nextTranslate = require('next-translate');
const nextConfig = {
  reactStrictMode: true,
  swcMinify: true,
};

module.exports = nextTranslate({ nextConfig });
`
package.json:
`"next-translate": "^1.5.0",`
any help please?

**Comment by aralroca** [comment, marked as answer]
First check if you have the locales folder with the jsons. If it's this case, I think is related with this issue https://github.com/aralroca/next-translate/issues/851 

The bug comes with SWC in Next.js 12, we need to investigate better to fix it. Any idea or proposal will be very welcome.

For now if you need to overwrite the `loadLocaleFrom` you can use this temporal workaround https://github.com/aralroca/next-translate/issues/851#issuecomment-1173611946
